gcc/rust/resolve/rust-ast-resolve-unused.h

```diff
@@ -35,8 +36,17 @@ class ScanUnused
       bool ok = r->lookup_canonical_path (decl_node_id, &ident);
       rust_assert (ok);
 
+      Analysis::Mappings *map = Analysis::Mappings::get ();
+      HirId decl_hir_id;
+      HIR::Item *finded_item = nullptr;
```

Comment:
I think you mean:

found_item instead of finded_item

Reply:
Oops, that's embarrassing.